src/connectdlg.cpp

```diff
@@ -112,14 +112,16 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
     // must not be specified since this column takes all the remaining space)
 #ifdef ANDROID
```

Comment:
Qt _should_ offer a platform independent way to get how much pixel space is taken up by a string.  If "that's too big generally", then we should then scale _all platforms_ back by the same multiplier.  (OK, not part of this PR, just noticing the nasty `#ifdef`.)

Reply:
I'll try to keep things like this in mind for the general Qt overhaul...